press/press/doctype/release_group/test_release_group.py

```diff
@@ -239,3 +240,94 @@ def test_cant_set_min_greater_than_max_workers(self):
 	def test_update_available_shows_for_first_deploy(self):
 		rg = create_test_release_group([create_test_app()])
 		self.assertEqual(deploy_information(rg.name).get("update_available"), True)
+
+	def test_fetch_environment_variables(self):
+		rg = create_test_release_group([create_test_app()])
+		frappe.local.team = lambda: frappe.get_doc("Team", rg.team)
```

Comment:
instead of overriding the team here we can specify `user=self.team.user` in `create_test_release_group` itself to assign the release group under the test team

Reply:
I tried out to use the team user, but while running in CI it's failing, it seems that the team in `self.team` and `frappe.local.team` is different.

Locally working fine !

Reply:
are you running tests on your developer instance or a separate test instance?

Reply:
I am running tests with this (ig as in site)
```
bench --site fc.local run-tests --doctype "Release Group"
```

Reply:
can you create a new site `fc.test` with press installed and run tests on it? (don't configure anything else, use it purely for testing)

Reply:
ok sure

Reply:
Run in a test site, that also passes all the testcase, team name also same.

![image](https://github.com/frappe/press/assets/57363826/4f554e57-bcbd-4b27-ab8a-4d1e16fe7517)

Reply:
I was checking fo this specific doctype, that's why it was not failing.

Found out the actual issue, in a specific testcase, the `frappe.local.team` was overwritten, that's why the new testcase was failing.

https://github.com/frappe/press/blob/84637d706b0ea76d367156ffb753124a1c2ad104/press/press/doctype/press_permission_group/test_press_permission_group.py#L125